tiptapy/__init__.py

```diff
@@ -89,7 +89,11 @@ def inner_render(self, node) -> str:
             caption = attrs.get('caption', '').strip()
             if caption:
                 html += f"<figcaption>{caption}</figcaption>"
-        return f'<div class="video-wrapper"><figure>{html}</figure></div>'
+        rendered_html = f'<div class="embed-wrapper link-wrapper"><figure>{html}</figure></div>'  # noqa: E501
+        provider_name = attrs.get('provider')
```

Comment:
@Pradhvan `provider = attrs.get('provider') or 'link'`
we can remove line 92

Reply:
@gauravr updated 😄